modules/ipi-install-network-requirements.adoc

```diff
@@ -26,25 +28,166 @@ When using a VLAN, each NIC must be on a separate VLAN corresponding to the appr
 ====
 
 [discrete]
-== Configuring the DNS server
+== DNS requirements
 
 Clients access the {product-title} cluster nodes over the `baremetal` network. A network administrator must configure a subdomain or subzone where the canonical name extension is the cluster name.
 
+[source,text]
 ----
 <cluster_name>.<domain>
 ----
 
 For example:
 
+[source,text]
 ----
 test-cluster.example.com
 ----
 
 {product-title} includes functionality that uses cluster membership information to generate A/AAAA records. This resolves the node names to their IP addresses. After the nodes are registered with the API, the cluster can disperse node information without using CoreDNS-mDNS. This eliminates the network traffic associated with multicast DNS.
 
+In {product-title} deployments, DNS name resolution is required for the following components:
+
+* The Kubernetes API
+* The {product-title} application wildcard ingress API
+
+Reverse DNS resolution is also required for the Kubernetes API, the provisioner node, the control plane nodes, and the worker nodes.
+
+A/AAAA records are used for name resolution and PTR records are used for reverse name resolution. {op-system-first} uses the reverse records or DHCP to set the hostnames for all the nodes. Additionally, the reverse records are used to generate the certificate signing requests (CSR) that {product-title} needs to operate.
```

Comment:
I'll have to double-check with the hardware deployment folks, but I believe the internal DNS server provides the necessary records to generate CSRs.

Reply:
CSRs are based on hostname, so the only DNS requirement is if reverse DNS is being used for hostname assignment. It's not a hard requirement. I think we could drop this sentence entirely since the hostname discussion in the previous sentence covers it.